examples/borrowed_window.rs

```diff
@@ -0,0 +1,47 @@
+//! Use borrowed window handles in `winit`.
```

Comment:
Could you just print it from the common window example, creating more examples to print just one thing seems sort of redundant.